components/TopBarProfileMenu.js

```diff
@@ -165,161 +185,169 @@ class TopBarProfileMenu extends React.Component {
         borderRadius="12px"
         boxShadow="0 4px 8px 0 rgba(61,82,102,0.08)"
         minWidth="170px"
-        maxWidth="300px"
+        maxWidth="500px"
         position="absolute"
         right={16}
-        top={40}
-        width="max-content"
+        top={75}
         zIndex={3000}
         data-cy="user-menu"
       >
-        <Container p={3}>
-          <Flex alignItems="center">
-            <P
-              color="#B4BBBF"
-              fontFamily="montserratlight, arial"
-              fontSize="1rem"
-              fontWeight="400"
-              letterSpacing="1px"
-              pr={2}
-              textTransform="uppercase"
-              whiteSpace="nowrap"
-            >
-              <FormattedMessage id="collective" defaultMessage="my collectives" />
-            </P>
-            <Container height="0.1rem" bg="#E6E6E6" width={1} minWidth={35} />
-            <Link route="/create" passHref>
-              <StyledLink buttonStyle="standard" buttonSize="tiny" display="inline-block" ml={2} whiteSpace="nowrap">
-                + <FormattedMessage id="Collective.New" defaultMessage="New" />
-              </StyledLink>
-            </Link>
-          </Flex>
-          <Box as="ul" p={0} my={2}>
-            {collectives.map(this.renderMembershipLine)}
-          </Box>
-          {collectives.length === 0 && (
-            <Box my={2}>
-              <P color="#9399A3" fontSize="1rem" letterSpacing="0.5px">
-                <em>
-                  <FormattedMessage id="menu.collective.none" defaultMessage="No collectives yet" />
-                </em>
+        <Flex flexWrap="wrap">
+          <Box order={[2, 2, 1]} width={[1, 1, 1 / 2]} p={[3]} bg="#F7F8FA">
+            <Hide xs sm>
+              <Avatar collective={get(LoggedInUser, 'collective')} radius="3rem" mr={2} />
```

Comment:
`LoggedInUser` should never be null in this component. We should be able to use it directly.

```suggestion
              <Avatar collective={LoggedInUser.collective} radius="3rem" mr={2} />
```

Reply:
Thanks. Done. :+1: